node/node.go

```diff
@@ -173,7 +173,7 @@ func (n *Node) Start() error {
 		MaxPendingPeers:  n.config.MaxPendingPeers,
 	}
 	running := &p2p.Server{Config: n.serverConfig}
-	log.Info(fmt.Sprint("instance:", n.serverConfig.Name))
+	log.Info("Starting peer-to-peer node", "instance", n.serverConfig.Name)
```

Comment:
I've just updated the log from the old format to the new style. I haven't finished going over the `node` package yet, all log messages there are still the auto-generated `fmt.Sprintf`. I just fixed this one in the PR since it revolved around the arch updates.